### PR TITLE
Adding laybuy icon for new payment app extension

### PR DESCRIPTION
**Pull request by scplaybuy**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by scplaybuy — COMMENTED** [review]
no brand link available - but verified by our marketing people - thanks.

**Review by scplaybuy — COMMENTED** [review]
updated from first test fail, removed extraneous file

**Comment by larouxn** [comment]
👋 Can we update the icon to match the rest of the icons (shape)? ([template](https://github.com/activemerchant/payment_icons/blob/bd0f7ce66eef2cac215145d466cc21cb3474e24c/CONTRIBUTING.md#template))
Screen Shot 2022-07-07 at 17 33 25
Screen Shot 2022-07-07 at 17 32 31

**Comment by scplaybuy** [comment]
@larouxn Hi Nicholas. I've checked the svg mark up for both visa and heart, and heart seems to be correct. Also in a svg view it appears to be the correct shape and size. opening in a browser probably isn't the best viewport? Can you let me know explicity what you want me to change as I can't see the issue :)

I've updated the image to include that template's example path elements too, if that's what you mean?
thanks,

see attached screen shorts
Screen Shot 2022-07-08 at 2 42 29 PM
Screen Shot 2022-07-08 at 2 42 41 PM

**Review by larouxn — CHANGES_REQUESTED** [review]
Hello, the icon is still not the correct shape. Maybe we can recreate it using the [template](https://github.com/activemerchant/payment_icons/blob/bd0f7ce66eef2cac215145d466cc21cb3474e24c/CONTRIBUTING.md#template)?
Screen Shot 2022-07-12 at 14 14 29

**Comment by scplaybuy** [comment]
Hi Nicholas,

Do you mean it's not got rounded corners? other icons have square corners, otherwise I can't see what you mean by not the same shape. I have the same height/width and viewport attributes as the template :)

please let me know explicitly what element of the shape you mean?

thanks,

**Comment by scplaybuy** [comment]
Have added rounded edges, assuming that's what you meant 👍🏼

**Comment by larouxn** [comment]
> Do you mean it's not got rounded corners? other icons have square corners, otherwise I can't see what you mean by not the same shape. I have the same height/width and viewport attributes as the template :)

Sorry for any confusion. All icons should look like the Visa icon I posted earlier. Any icon that's square/a different shape, is probably old and/or before we had icon guidelines.

**Comment by scplaybuy** [comment]
Hi,
I used the template. the only different is the background colour and vectors that create the laybuy shape :)

File: db/payment_icons.yml [hunk 1]
```diff
@@ -1358,3 +1358,7 @@
   name: bsi
   label: BSI
   group: bank_transfers
+-
+  name: heart
```
Comment (larouxn) [inline db/payment_icons.yml:1362]: Can we rebase this PR and fix this name? Should be `laybuy`.

**Comment by scplaybuy** [comment]
ive renamed the old one laybuy-gateway.. and the new one (heart) to laybuy.
I can't see anything in dev note but I assume these names = app name used?

**Comment by larouxn** [comment]
> ive renamed the old one laybuy-gateway.. and the new one (heart) to laybuy. I can't see anything in dev note but I assume these names = app name used?

Should all places using the old icon start seeing the new icon automatically? If not then maybe we should actually keep the old icon as `laybuy` and change the new one to `laybuy-heart`.

**Comment by scplaybuy** [comment]
hi,
i really don't know how their icon system joins up - i think now that the gateway is defunct and all merchants should be using the new app (https://partners.shopify.com/616717/apps/5822183/overview#/) - what we want is this icon to appear next to Laybuy when they app is installed and customers see Laybuy as a payment option at checkout.

**Comment by scplaybuy** [comment]
next to the text Laybuy at checkout: [](url
Screen Shot 2022-07-21 at 6 41 08 PM
)

**Comment by larouxn** [comment]
I see now. It looks like `Laybuy` was rebranded from the old icon (currently in this repo) to the new one (added in this PR). In that case, do you want to remove the old icon in this PR too? We can just completely replace the old icon with the new icon.

Screen Shot 2022-07-21 at 15 55 25
Screen Shot 2022-07-21 at 15 55 32

In summary
- remove old Laybuy icon
- add new Laybuy icon with same name
- update `payment_icons.yml` entry (looks `label` is different now)

**Comment by scplaybuy** [comment]
Hi Nicolas,
I've checked with our product expert, that older laybuy icon is actually a totally different company.
So, I'll change our one to laybuy-heart if that works for you and push again, reverting the old icon name change.
thanks, simon

**Comment by larouxn** [comment]
> Hi Nicolas, I've checked with our product expert, that older laybuy icon is actually a totally different company. So, I'll change our one to laybuy_heart if that works for you and push again, reverting the old icon name change. thanks, simon

Oh wow, okay. Interesting. That works for me. Thanks!

**Review by larouxn — DISMISSED** [review]
Looks good to me 🎩 
Screen Shot 2022-07-26 at 16 11 06

**Comment by larouxn** [comment]
Need to update the SVG with the new ID: `pi-heart` → `pi-laybuy_heart`. ([CI logs](https://github.com/activemerchant/payment_icons/runs/7515012714?check_suite_focus=true))

```ruby
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:81]:
{:message=>"The 'laybuy_heart' SVG file does not have the appropriate 'id' value on the <title> tag"}.
Expected: "pi-laybuy_heart"
  Actual: "pi-heart"

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

..........

Finished in 1.057246s, 12.2961 runs/s, 5817.9443 assertions/s.
13 runs, 6[15](https://github.com/activemerchant/payment_icons/runs/7515012714?check_suite_focus=true#step:4:16)1 assertions, 1 failures, 0 errors, 0 skips
```

**Comment by AlexRohloff** [comment]
Nicholas, Simon is currently on leave. Do you have everything you need here and can we get an ETA on when the logo will be live?

**Comment by larouxn** [comment]
> Nicholas, Simon is currently on leave. Do you have everything you need here and can we get an ETA on when the logo will be live?

Hey Alex, we need to update the SVG to use `laybuy-heart` for it's IDs re: https://github.com/activemerchant/payment_icons/pull/675#issuecomment-1195096695.

**Comment by AlexRohloff** [comment]
> > Nicholas, Simon is currently on leave. Do you have everything you need here and can we get an ETA on when the logo will be live?
> 
> Hey Alex, we need to update the SVG to use `laybuy-heart` for it's IDs re: [#675 (comment)](https://github.com/activemerchant/payment_icons/pull/675#issuecomment-1195096695).

Is this something we need to do or managed at your end?

**Comment by larouxn** [comment]
> > > Nicholas, Simon is currently on leave. Do you have everything you need here and can we get an ETA on when the logo will be live?
> > 
> > 
> > Hey Alex, we need to update the SVG to use `laybuy-heart` for it's IDs re: [#675 (comment)](https://github.com/activemerchant/payment_icons/pull/675#issuecomment-1195096695).
> 
> Is this something we need to do or managed at your end?

Need to do it on your end, here in this PR or CI will continue failing and we won't be able to merge.

https://github.com/activemerchant/payment_icons/runs/7515012714?check_suite_focus=true

**Comment by scplaybuy** [comment]
@larouxn updated

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-08-01 at 13 40 45
Screen Shot 2022-08-01 at 13 41 17

@tauthomas01, what do you think?

**Comment by larouxn** [comment]
Hey @scplaybuy, nothing else you need to do on your end. We'll get this merged and released, then I'll comment back here when it's live/ready to be used.

**Review by tauthomas01 — APPROVED** [review]
Looks good

**Comment by larouxn** [comment]
👋 Hello, we're working on the [v1.7.16](https://github.com/activemerchant/payment_icons/pull/702) release now. It will likely go live next week.

Apologies for the wait. Will report back here when it's live. 🙇‍♂️

**Comment by todd-lb** [comment]
Kia ora @larouxn, are you able to please provide an update on this please?

**Comment by larouxn** [comment]
👋 Hello, your new/updated icon(s) should now be live everywhere. ✨

**Comment by scplaybuy** [comment]
Hi Nicholas,

Do you mean it's not got rounded corners? other icons have square corners,
otherwise I can't see what you mean by not the same shape. I have the same
height/width and viewport attributes as the template :)

please let me know explicitly what element of the shape you mean?

thanks,

Simon

**Pull request merged**